### DISCUSSION TITLE
Transcript - Sentence are repeating

### DISCUSSION THREAD

**Discussion by Arasun01** [SYSTRAN/faster-whisper#434] [Q&A] [answered]
I am using the following code to get the transcripts, I am seeing some repeated text. Do you have any idea how to avoid the repetitive text? Please recommend

Code:

from faster_whisper import WhisperModel
model_size = "large-v2"

# Run on GPU with FP16
model = WhisperModel(model_size, device="cuda", compute_type="float16")
audio_file = "audio1.m4a"

segments, _ = model.transcribe(audio_file, temperature=(0.0, 0.2, 0.4, 0.8, 1.0),
                best_of=5, 
                beam_size=5, 
                suppress_tokens=[-1], 
                condition_on_previous_text=True,
                compression_ratio_threshold=2.4, 
                log_prob_threshold=-1., 
                no_speech_threshold=0.3,
                word_timestamps=True)

Output:

Okay, for Biggie, his exam findings, very healthy 10 year old, looks great. NSF through the whole thing. His mouth, I would say calc level 1 and a few missing teeth from prior extractions, TPR is 100.3, 130, panting, CRT less than 2, CRT less than 2, pink, 0 pain, body condition score of 6 and 35.6 pounds. His wellness, or health status is fair, good and stable. His wellness concern, health assessment is healthy, geriatric, canine well template for the plan. Today, we're doing an acuplex, we're doing a rabies and distemper 3 year vaccine, we're going to recommend dental cleaning and dental care at home, weight loss reduced, daily calorie intake and we're going to do an acuplex, we're doing a rabies and distemper 3 year vaccine, we're going to recommend dental cleaning and dental care at home, weight loss reduced, daily calorie intake and we're going to do an acuplex, we're going to do an acuplex, we're doing a rabies and distemper 3 year vaccine, we're going to recommend dental cleaning and dental care at home, weight loss reduced, daily calorie intake and **we're going to do an acuplex, we're going to do an acuplex, we're going to do an acuplex, we're going to do an acuplex, we're going to do an acuple**x, we're going to discuss YouMove advanced as a joint supplement to help with mobility and perhaps counteract the sleeping, increased sleeping that the owner has noticed. Slight language barrier discussed or oral and phlegm preventive

**Comment by phineas-pta** [comment]
try with `vad_filter=True`

**Comment by Arasun01** [comment, marked as answer]
Yes, Thanks @Phan Tuan Anh. I fixed this morning with that parameter. It
works now.
